File: backend/Metropolitan/Common/model/flujoCaja.cs
using System;

namespace Common.model
{
    public class FlujoCajaResumen
    {
        public string cuenta;
        public string moneda;
        public double totalIngresos;
        public int cantidadOperaciones;
    }

    public class FlujoCajaMovimiento
    {
        public DateTime fechaPago;
        public string modulo; // TRAVELACE | PAQUETES | CARGA
        public int idOrdenPago;
        public int idNotaDebito;
        public string cuenta;
        public string formaPago;
        public string moneda;
        public double monto;
        public string concepto;
    }
}

File: frontend/metropolitan/src/app/shared/model/flujo-caja.ts
export interface FlujoCajaResumen {
  cuenta: string;
  moneda: string;
  totalIngresos: number;
  cantidadOperaciones: number;
}

export interface FlujoCajaMovimiento {
  fechaPago: string;
  modulo: string;
  idOrdenPago: number;
  idNotaDebito: number;
  cuenta: string;
  formaPago: string;
  moneda: string;
  monto: number;
  concepto: string;
  tipoCambioValor?: number;
}
